testdata/scripts/keys/eth/list/unavailable.txtar

```diff
@@ -0,0 +1,38 @@
+# start node
+exec sh -c 'eval "echo \"$(cat config.toml.tmpl)\" > config.toml"'
+exec chainlink node -c config.toml start -p password -a creds &
+
+# initialize client
+env NODEURL=http://localhost:$PORT
+exec curl --retry 10 --retry-max-time 60 --retry-connrefused $NODEURL
+exec chainlink --remote-node-url $NODEURL admin login -file creds --bypass-version-check
+
+exec chainlink --remote-node-url $NODEURL keys eth list
+! stdout 'ETH:               <nil>'
+! stdout 'LINK:              <nil>'
```

Comment:
These are grep strings, so there is probably a cleaner, more flexible way to check for these... :thinking: